includes/Modules/Sign_In_With_Google.php

```diff
@@ -409,25 +444,36 @@ private function render_signinwithgoogle() {
 		library_name: 'Site-Kit'
 	} );
 
-	<?php if ( $render_buttons ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
-		const parent = document.createElement( 'div' );
+	const parent = document.createElement( 'div' );
 
-		<?php if ( $is_wp_login ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
-			document.getElementById( 'login' ).insertBefore( parent, document.getElementById( 'loginform' ) );
-		<?php endif; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
+	<?php if ( $is_wp_login ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
+		document.getElementById( 'login' ).insertBefore( parent, document.getElementById( 'loginform' ) );
+	<?php endif; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
 
-		<?php if ( $is_woocommerce_login ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
-			parent.classList.add( 'woocommerce-form-row', 'form-row' );
-			const form = document.querySelector( '.woocommerce-form.login' );
-			if ( form ) {
-				form.insertBefore( parent, form.firstChild );
-			}
-		<?php endif; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
+	<?php if ( $is_woocommerce_login ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
+		parent.classList.add( 'woocommerce-form-row', 'form-row' );
+		const form = document.querySelector( '.woocommerce-form.login' );
+		if ( form ) {
+			form.insertBefore( parent, form.firstChild );
+		}
+	<?php endif; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
 
-		google.accounts.id.renderButton( parent, <?php echo wp_json_encode( $btn_args ); ?> );
+	google.accounts.id.renderButton( parent, <?php echo wp_json_encode( $btn_args ); ?> );
+
+	<?php if ( ! is_user_logged_in() ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
+		/**
+		 * Render SiwG buttons for all `<div>` elements with the "magic class"
+		 * on the page.
+		 *
+		 * Mainly used by Gutenberg blocks.
+		 */
```

Comment:
Let's make this comment to be a php comment to avoid it appearing in the output.